Scripts/Editor/MeshChainShaderGUI.cs

```diff
@@ -139,10 +139,10 @@ public void ShaderPropertiesGUI(Material material)
                 var newRadiusScale = EditorGUILayout.FloatField(Styles.lineRadiusScaleText, radiusSettings.x);
                 var newRadiusMin = radiusSettings.y;
                 var newRadiusMax = radiusSettings.z;
+                newRadiusMin = Mathf.Max(EditorGUILayout.FloatField(Styles.lineRadiusMinText, radiusSettings.y), 0.0f);
```

Comment:
With this outside the if block now, we don't need `newRadiusMin = radiusSettings.y` above.